server/routes/tracker.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+
+const trackerController = require('./controllers/tracker.controller');
+const authentication = require('./config/authentication');
```

Comment:
dont have this

Reply:
I wrote it because I needed authentication of user for my function but this file wasn't written before. 
I have fixed for the latest
const authentication = require('./middleware/authRequire');